config/settings.yml

```diff
@@ -32,3 +32,5 @@ workflow:
 
 redis:
   url: ~
+
+purl_url: 'https://purl-example.stanford.edu'
```

Comment:
Has this change been made in shared configs for the 3 environments?

Reply:
Yes.